1_Diversity-Participation.md

```diff
@@ -0,0 +1,28 @@
+# Metric: Diversity and Participation
+
+Activity Metric | Description
+--- | ---
+New Contributors vs Maintainers | Ratio of new contributors to maintainers over time
+Contributor Demographics | Gender, age, location, education, and skills over time
+New Contributor Corporate Affiliations | New corporate affiliations over time
+New Contributions | Percentage of contributions (patches, pull requests, etc.) from new contributors vs all accepted contributions over time
+Accepted Code Contributions | Percentage of new contributor code versus total code over time
+Path to Maintainership | Path to maintainership communicated

```

Comment:
Could we have an example of this, or maybe a short explanation?

Reply:
Path to Maintainership = A governance document that describes the criteria and process by which a contributor can be promoted to maintainer (i.e. be granted commit rights).

Reply:
Thanks for the example. Maybe the metric is therefore a binary metric, something like "Existence of a public document detailing the path to mantainership"?

Reply:
That is a better description indeed.